src/hotspot/share/gc/g1/g1RemSet.cpp

```diff
@@ -1078,10 +1078,12 @@ class G1MergeHeapRootsPrefetchCache {
   NONCOPYABLE(G1MergeHeapRootsPrefetchCache);
 
 protected:
-  // Initial content of all elements in the cache. It's value be "neutral", i.e.
-  // no work done on it.
+  // Initial content of all elements in the cache. It's value should be
+  // "neutral", i.e. no work done on it when processing it.
   G1CardTable::CardValue _dummy_card;
 
+  ~G1MergeHeapRootsPrefetchCache() { }
```

Comment:
Better would be a defaulted definition, i.e. `~G1MergeHeapRootsPrefetchCache() = default;`